docs/architecture/microservices/implement-resilient-applications/implement-circuit-breaker-pattern.md

```diff
@@ -93,7 +93,7 @@ You can then check the status using the URI `http://localhost:5103/failing`, as
 
 **Figure 8-5**. Checking the state of the "Failing" ASP.NET middleware – In this case, disabled.
 
-At this point, the Basket microservice responds with status code 500 whenever you call invoke it.
+At this point, the Basket microservice responds with status code 500 whenever your call invokes it.
```

Comment:
Probably it was meant to be a "slash" or "or" between "call" and "invoke".

```suggestion
At this point, the Basket microservice responds with status code 500 whenever you call or invoke it.
```

Reply:
Okay, that makes perfect sense to me. I opted for a slash since that's pretty much what's used across the doc.
Thank you @Youssef1313